monkey/tests/unit_tests/monkey_island/cc/resources/test_clear_simulation_data.py

```diff
@@ -0,0 +1,29 @@
+from http import HTTPStatus
+from unittest.mock import MagicMock
+
+import pytest
+from tests.common import StubDIContainer
+
+from monkey_island.cc.event_queue import IIslandEventQueue, IslandEventTopic
+from monkey_island.cc.resources import ClearSimulationData
+
+
+@pytest.fixture
+def mock_island_event_queue() -> IIslandEventQueue:
+    return MagicMock(spec=IIslandEventQueue)
+
+
+@pytest.fixture
+def flask_client(build_flask_client, mock_island_event_queue):
+    container = StubDIContainer()
+    container.register_instance(IIslandEventQueue, mock_island_event_queue)
+
+    with build_flask_client(container) as flask_client:
+        yield flask_client
+
+
+def test_clear_simulation_data(flask_client, mock_island_event_queue):
+    resp = flask_client.post(ClearSimulationData.urls[0], follow_redirects=True)
```

Comment:
Why do we need `follow_redirects=True`?

Reply:
@mssalvatore

Reply:
We have it everywhere and I haven't investigated it. For now, consistency. If someone wants to figure out why we use it everywhere, be my guest.